lib/types/index.js

```diff
@@ -953,6 +959,7 @@ export type UserProfile = {
   name: string,
   nickname: string,
   picture: string,
+  sub: string,
```

Comment:
Why was this added?

Reply:
This field contains the user id in profiles returned by `useAuth0`.